app/jobs/calendar_importer/events/base.rb

```diff
@@ -27,7 +27,7 @@ def sanitize_invalid_char(input)
     # Convert h1 and h2 to h3
     # Strip out all shady tags
     # Convert all html to markdown
-    def html_sanitize(input)
+    def html_sanitize(input, just_as_text: false)
```

Comment:
Can we give this argument a more descriptive name, please? For example:

```suggestion
    def html_sanitize(input, convert_to_plaintext: false)
```

Reply:
Thought actually, thinking about it, why would we ever want the old behaviour? Can we just replace the old behaviour with the much simpler and more reliable new behaviour?

Reply:
Some people may have HTML in their text fields (via a JSON or XML export API) and we need to preserve that document structure. The summary field should be basic plaintext but you can never rely on user provided data to be sane. This switch allows the old behaviour to do a full clean but also allows us to "just" get plaintext

Reply:
fair enough